cmake/horovod/CMakeLists.txt

```diff
@@ -6,7 +6,7 @@ set(CMAKE_CXX_STANDARD 11)
 
 set(THIRD_PARTY_ROOT ${HOROVOD_ROOT}/third_party)
 
-set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -pthread -B /data/anaconda/envs/ort/compiler_compat -Wl,--sysroot=/ -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -fPIC -DHAVE_MPI")
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -pthread -Wl,--sysroot=/ -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -fPIC -DHAVE_MPI")
 
 set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wno-error=strict-aliasing -Wno-error=unused-parameter -Wno-error=ignored-qualifiers -Wno-error=maybe-uninitialized")
```

Comment:
Please don't change CMAKE_CXX_FLAGS.

Use "target_compile_options(horovod PRIVATE" instead. 

Please do not put "-DNDEBUG" there. It will mess up debug build.